consensus/src/marshal/ingress/mailbox.rs

```diff
@@ -3,32 +3,61 @@ use crate::{
     types::Round,
     Block, Reporter,
 };
-use commonware_cryptography::bls12381::primitives::variant::Variant;
+use commonware_cryptography::{bls12381::primitives::variant::Variant, Digest};
+use commonware_storage::archive;
 use futures::{
     channel::{mpsc, oneshot},
     SinkExt,
 };
 use tracing::error;
 
+/// An identifier for a block request.
+pub enum Identifier<D: Digest> {
+    /// The height of the block to retrieve.
+    Height(u64),
+    /// The commitment of the block to retrieve.
+    Commitment(D),
+    /// The highest finalized block. It may be the case that marshal does not have some of the
+    /// blocks below this height.
+    Tip,
```

Comment:
nit: for consistency prefer `Latest` (although I suspect the rational for `Tip` was that `Latest` is better suited for the last processed block)?

Maybe we do both?

Reply:
Naming is hard, I think `Latest` is fine, I will hold off from doing `LastProcessed` at the moment since:
- it's not totally clear that it's needed yet
- seems pretty easy to add later
- I don't want to over-promise on a type that could possibly later be removed

Reply:
👍